### DISCUSSION TITLE
Can I add spaces in kubebuilder comments?

### DISCUSSION THREAD

**Discussion by mateusoliveira43** [kubernetes-sigs/kubebuilder#3801] [Support] [answered]
When setting up linters to some projects, I saw the following error:
```
cmd/main.go:52:2: comment-spacings: no space between comment delimiter and comment text (revive)
	//+kubebuilder:scaffold:scheme
	^
```

Can I safely add a space between `//` and `+` to fix these errors or this can break my usage of kubebuilder?

In the kubebuilder book, I saw comments in both formats, i.e., `//+kubebuilder` and `// +kubebuilder`.

**Comment by camilamacedo86** [comment, marked as answer]
Hi @mateusoliveira43 

Thank you for raise this one.
I agree that we should have an standard and ensure that all places are in the same.
Currently, the markers scaffold by Kubebuilder CLI are without space see: https://github.com/kubernetes-sigs/kubebuilder/blob/2b25adec1ffb8e811a85e8d86357adcf7213bf5b/pkg/machinery/marker.go#L57

However, we have also controller-tools and seems that there all are with spaces: https://github.com/search?q=repo%3Akubernetes-sigs%2Fcontroller-tools%20%2F%2F%20%2Bkubebuilder%3A&type=code


Due the linter issue raised by you, I think we can ensure that we have an space always, see: https://github.com/kubernetes-sigs/kubebuilder/pull/3904

fell free to review this one.

**Comment by camilamacedo86** [comment]
We opened an PR to change it out, feel free to review.
I am closing this one since we have a PR.
